### DISCUSSION TITLE
[Feature] Text editor option for JSON

### DISCUSSION THREAD

**Discussion by kyleplo** [bridge-core/bridge.#250] [Ideas]
**Summary**
Bridge. features a folding JSON editor for editing JSON code. This is useful in some cases, but sometimes a text editor is nicer.

**Alternatives/Considerations**
An option to switch between text and folding editors for JSON. It would be nice if the text editor still had syntax highlighting and autocomplete.

**Comment by solvedDev** [comment]
This feature is on our roadmap but it requires some changes to the app architecture first 👍
